app/models/legislation.rb

```diff
@@ -1,3 +1,18 @@
+# == Schema Information
+#
+# Table name: legislations
```

Comment:
I think you need to rebase against `develop`. Not sure why github is showing those changes.

Reply:
I think that is what I did.

Reply:
or maybe not...

Reply:
done :+1: